### ISSUE TITLE
[Request] Export video with overlay already applied?

### ISSUE THREAD

**Issue by ketxxx** [cirquit/trdrop#147]
As there isn't a request section I guess it has to go here. I'm wondering if there are any plans to make it so you can export a video with the FPS analysis overlayed to the video. Otherwise the only way to do things is the long way either through video editing or capturing the capture as the software exports... both of which are kinda a pain in the arse.

**Comment by cheapest-energy-windows** [comment]
you can already do this, export as jpeg and make sure "Export as overlay" is not enabled, use your editing software or ffmpeg to create a video out of the jpegs

**Comment by ketxxx** [comment]
Thanks but that's not quite what I'm asking, having to run a video though several pieces of software kinda defeats the purpose of TrDrop as a tool. I'm asking if there's an option I've missed or a planned option to enable TrDrop to directly export videos with the framerate analysis already overlayed ;) Having to individually generate over 9000 images (in the case of my test video of 6 mins) then having to use another piece of software to stitch everything together is a very long winded and messy way to do things :(

**Comment by cheapest-energy-windows** [comment]
Yes I understand, I have it scripted. When te output is ready I just run the script (ffmpeg) and make a prores with alpha video that I can just use as overlay (I export it as .tiff). You could do the same with the jpegs and have it stitched together as mp4. Still not what you really want but something like this might sooth the pain.

**Comment by ketxxx** [comment]
Thanks for the follow-up, sadly, it looks like I'm going to have to keep searching which is a real shame I like how TrDrop works being a quick and straightforward UI but not being able to directly export a video is painful, particularly as I'm literally sitting there watching the software analyse the framerate of the video in realtime when creating all the images.
